src/workerd/api/streams/common.h

```diff
@@ -287,12 +287,12 @@ class ReadableStreamSource {
 };
 
 struct PipeToOptions {
-  jsg::Optional<bool> preventClose;
   jsg::Optional<bool> preventAbort;
   jsg::Optional<bool> preventCancel;
+  jsg::Optional<bool> preventClose;
```

Comment:
If the ordering of these is important, then it would be worth adding comments to discourage any reordering in the future... here an with the `writable`/`readable` below.